yolov5 classification question, AssertionError: Label class 2.0 exceeds nc=2 in ./data/custom_data.yaml. Possible class labels are 0-1

## ❔Question

## Additional context
Hello, I have limited label for 0 or 1 in all samples,but i still encounter this problem during training, could you help me?
    assert mlc < nc, f'Label class {mlc} exceeds nc={nc} in {data}. Possible class labels are 0-{nc - 1}'
AssertionError: Label class 2.0 exceeds nc=2 in ./data/custom_data.yaml. Possible class labels are 0-1

--- a/train.py
+++ b/train.py
@@ -204,7 +204,7 @@ def train(hyp,  # path/to/hyp.yaml or hyp dictionary
                                               hyp=hyp, augment=True, cache=opt.cache, rect=opt.rect, rank=RANK,
                                               workers=workers, image_weights=opt.image_weights, quad=opt.quad,
                                               prefix=colorstr('train: '))
-    mlc = np.concatenate(dataset.labels, 0)[:, 0].max()  # max label class
+    mlc = int(np.concatenate(dataset.labels, 0)[:, 0].max())  # max label class
     nb = len(train_loader)  # number of batches
     assert mlc < nc, f'Label class {mlc} exceeds nc={nc} in {data}. Possible class labels are 0-{nc - 1}'
 

